client/e2e/add-condition.spec.ts

```diff
@@ -1,39 +1,12 @@
 import { test, expect } from './fixtures/fixtures';
-import { login, logout } from './utils';
-import { CONFIGURATION_CTA } from '../src/pages/Configurations/utils';
-
-test.describe
-  .serial('Adding/modifying configurations by initial condition', () => {
-  test.beforeEach(async ({ page }) => {
-    await login(page);
-  });
-
-  test.afterEach(async ({ page }) => {
-    await logout(page);
-  });
+import { createNewConfiguration } from './utils';
 
+test.describe('Adding/modifying configurations by initial condition', () => {
   test('should be able to create a configuration for Acanthamoeba', async ({
     page,
     makeAxeBuilder,
   }) => {
-    /// ==========================================================================
-    /// Test that a new condition can be added
-    /// ==========================================================================
-    await page.getByRole('button', { name: CONFIGURATION_CTA }).click();
-
-    await expect(makeAxeBuilder).toHaveNoAxeViolations();
-
-    await page.getByTestId('combo-box-input').click();
-    await page.getByTestId('combo-box-input').fill('Acanthamoeba');
-    await page.getByTestId('combo-box-input').press('Tab');
-    await page.getByRole('option', { name: 'Acanthamoeba' }).press('Enter');
-    await page.getByTestId('combo-box-input').press('Tab');
-    await page.getByTestId('combo-box-clear-button').press('Tab');
-    await page.getByTestId('modalFooter').getByTestId('button').click();
-    await expect(
-      page.getByRole('heading', { name: 'New configuration created' })
-    ).toBeVisible();
-
+    await createNewConfiguration('Acanthamoeba', page);
```

Comment:
after this gets in, going to wrap this into some sort of fixture that does the setup automatically, potentially at the worker level, so that the tests are more parallellizable. That way each of the tests can have a more ergonomic way of getting a clean config to be working off of